README.md

```diff
@@ -99,3 +99,54 @@ poetry publish
 Check the [PyPi Release history](https://pypi.org/project/dbt-platform-helper/#history) to make sure the package has been updated.
 
 For an optional manual check, install the package locally and test everything works as expected.
+
+## Releases
+
+### Release structure
+
+- Use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) in our pull request titles. These become the release commit messages in the `main` branch. The advantage of this is that it's clear from a list of commits, what kind of changes they include (_fix=patch, feat=minor, !=breaking change_) and most importantly whether commits include breaking changes
+- Use [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) to document and manage releases
+- Non-breaking changes already in the `main` branch should be released before merging a breaking change
+- Minor and patch releases should always include zero downtime
+- Documentation for releases with breaking changes should include the upgrade path
+- Where possible the upgrade path should allow for zero downtime, if this is not possible, it should be flagged up big and visible
+- Pull the release information from GitHub Releases into the DBT Platform Documentation (currently WIP)
+- Clicking "Publish release" in GitHub should automatically:
+  - Run the full regression pipeline (currently WIP)
+  - Automate version bumping based on the types in the commit messages (currently WIP)
+  - Publish the package to PyPI
+  - Trigger a rebuild of the DBT Platform Documentation, so it includes the latest release documentation (currently WIP)
+  - Push a notification to the development community via the #developers channel in Slack
+
+### Release procedure - Manual steps
+
+- Create a new branch
+- Ensure you have incremented the `dbt-platform-helper` package version in the root _pyproject.toml_ file. This new version will be released to PyPi and therefore should not already exist as a release version in [PyPI release history](https://pypi.org/project/dbt-platform-helper/#history)
+- During the release process, we run both unit tests and regression tests
+- Create a PR and have it reviewed
+- On approval, merge to main
+- This will trigger a codebuild project called `platform-tools-test` in the _platform-tools_ AWS account to run. 
```

Comment:
```suggestion
- This will trigger a CodeBuild project called `platform-tools-test` in the _platform-tools_ AWS account to run regression tests on `merge to main / pull request created / pull request updated` events emitted by Github
```